README.md

```diff
@@ -2,24 +2,251 @@
+
+* For each release, lead the establishment of release criteria by the ASC. This can include a list of proposed modifications to be decided upon prior to release and target release dates
+* Shepherd progress towards completing the schedule through interactions with proposal authors
+* Maintain the Revisions section of the document to ensure accurate communication of what has changed in each release
+
+Note that Release Managers are not precluded from proposing modifications to the Standard (including to the series under their management) during the course of their assignment. In addition, Release Managers are not responsible for promoting acceptance of any proposal nor contributing to the completion of a proposal in order to meet schedule, and may resolve any merge conflicts resulting from commit of approved changes without further process.
+
+The Release Managers shall notify the Co-Chairs when a minor release candidate is complete and ready for publication. The Co-Chairs will add an item to the agenda of the next quarterly meeting (subject to the one month deadline) for final approval to publish the document. Once the ASC has voted its approval, the Release Managers shall post the new release on the community's web site and send an email announcing the release to the community mailing list. Minor releases, therefore, can occur on an at most a quarterly basis.
+
+Major release candidates must be approved during the ASC's annual face-to-face meeting. Release Managers must notify the Co-Chairs of a major release candidate a minimum of two quarters before the target approval meeting. The Co-Chairs will add an item to the agenda of each quarterly meeting for discussion and review of the document, and on the face-to-face meeting for vote on final approval. Once the ASC has voted its approval, the Release Managers shall post the new release on the community's web site and send an email announcing the release to the community mailing list. Should the ASC not approve the release, subsequent release candidates can be considered at upcoming regular ASC meetings - i.e., the major release candidate does not have to wait for the next annual face-to-face meeting for reconsideration. However, no subsequent major release may begin the approval process until after the current candidate has been published. Major releases, therefore, can occur no more than once per year.
+
+
+## The Standardization Process
+
+Contributions of any form (e.g., use cases, questions, suggestions) are encouraged from anyone interested in the evolution of the PMIx Standard. The community relies on Issues and Pull Requests to discuss changes to the standard for a few reasons:
+
+* It provides an archive of the conversation around a change to the
```

Comment:
I would maybe avoid specifying Git concepts in this document.  I know we all love git and think it will be here forever, but this is like the 5th revision control system I've used and may not be the last.   Maybe the next one won't have pull requests.

Reply:
Wow - I don't see any way to do that without creating total confusion. If the community migrates to some other mechanism in the future, you'll need to modify the governance to reflect the newly revised mechanism.

Reply:
Perhaps it could be accomplished by separating out the general steps for standardization from the actual implementation of them. What I mean is, you could talk in general terms about creating a proposal that is in a "draft" state and how that proposal moves from draft to reading to provisional, etc. Then you can refer to a "workflow document" or something like that that outlines how we would use github to manage the proposals. Then later when github is replaced by something else, the changes would be compartmentalized. 

All that said, I am not sure which approach is better. You need to tell people exactly what to do somewhere...

Reply:
Yeah, I can somewhat picture what you are saying - I'm just concerned that it gets too fragmented. In other words, someone has to read this writeup, then go read the workflow writeup, then try to piece it all together. Seems a little imposing and prone to lots of "but how do I" questions.

Reply:
If it would be helpful to potential contributors, we can add a `CONTRIBUTING` document that summarizes the contributing steps (with lots of details about Github) and elides the governance details.